### ISSUE TITLE
ECMAScript 2017 HTML version - search broken, various 404s and script errors

### ISSUE THREAD

**Issue by farsightsoftware** [tc39/ecma262#959]
The search box in the HTML version of the latest released spec, http://www.ecma-international.org/ecma-262/8.0/index.html, is broken (typing something, say "class", doesn't find it). There are various 404s and script errors as you use the page.

Looks like a deployment issue on the ECMA site, these issues don't happen at https://tc39.github.io/ecma262/.

404s:

* http://www.ecma-international.org/ecma-262/8.0/ECMAScript%C2%AE%202017%20Language%20Specification_files/ecmarkup_002.htm
* http://www.ecma-international.org/ecma-262/8.0/ECMAScript%C2%AE%202017%20Language%20Specification_files/github.css
* http://www.ecma-international.org/ecma-262/8.0/ECMAScript%C2%AE%202017%20Language%20Specification_files/ecmarkup.htm

There's also an unexpected token error:

```
index.html:23 Uncaught SyntaxError: Unexpected token <
```

Presumably as a result of the 404s, mousing over various things causes errors like this:

```
index.html:685 Uncaught TypeError: Cannot read property
'prod-grammar-notation-StatementList' of undefined
    at Object.activateIfMouseOver (index.html:695)
    at HTMLDivElement.<anonymous> (index.html:501)
    at HTMLDivElement.<anonymous> (index.html:520)
```

(I [posted this on es-discuss](https://esdiscuss.org/topic/ecmascript-2017-html-version-search-broken-various-404s-and-script-errors) several days ago, but the suggestion was to open an issue here.)

**Comment by IgnoredAmbience** [comment]
A couple of other issues:
* An erroneous (and unclosed?) `<script>` tag has been inserted above the `<title>` tag.
* The favicon URL is unstable, pointing to `https://tc39.github.io/ecma262/2017/img/favicon.ico`. The 2017 directory is likely to disappear when 2018 is progressed.
* There's a few absolute links pointing to the draft standard, instead of being relative to the current document. (grep the html source for `tc39.github.io/ecma262/#`) (these seem to persist in the original source though, so I'll raise a PR as appropriate for this.)

The 404ing URLs appear to be referencing the directory created by a browser's "Save Page" functionality (complete with the &copy; symbol from the page title).

**Comment by bterlson** [comment]
I will look in to this, thank you!

**Comment by tjcrowder** [comment]
Bump. I refer people to the specification all the time. Would be better if it were working properly.

**Comment by domenic** [comment]
Don't refer them to the outdated snapshot on that website. Refer them to the one implemented by browsers at https://tc39.github.io/ecma262/ .

**Comment by tjcrowder** [comment]
@domenic - Well, implemented by varying browsers at varying times. That's beside the point. The most recent snapshot should work properly.

**Comment by domenic** [comment]
No, ecma262 only includes things which have reached stage 4, and are implemented in multiple browsers.

The snapshot should generally not be referred to, especially by programmers.

**Comment by tjcrowder** [comment]
@domenic - Useful to know (where is that documented?). It's still beside the point. The snapshot should work properly, and it's perfectly valid to need to refer someone to how things were as of ES2017.

**Comment by domenic** [comment]
I'll just leave it as "I disagree"; I think the less effort people spend on the snapshot, the better, so as to discourage people from referring to it. This is just _my_ opinion, but I do think it'd be good to tone down your expectation that other people do work for you to fit _your_ opinion.

**Comment by tjcrowder** [comment]
@domenic I don't deserve that slap. I've offered to help in the past and would happily help with this if it were something I could help with.

**Comment by ljharb** [comment]
@patrickcharollais there's some issues with the 2017 edition of the spec on ecma's website; it'd be great if you could fix them (or state your refusal to do so) so we can close this issue.
